hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TopicClient.java

```diff
@@ -117,8 +117,9 @@ public TransactionReceipt deleteTopic(ConsensusTopicId topicId) throws HederaSta
         log.debug("Publishing {} message(s) to topicId : {}.", numMessages, topicId);
         List<TransactionReceipt> transactionReceiptList = new ArrayList<>();
         for (int i = 0; i < numMessages; i++) {
-            byte[] byteArray = Longs.toByteArray(Instant.now().toEpochMilli());
-            String message = Base64.encodeBase64String(byteArray) + "_" + baseMessage + "_" + i + 1;
+            byte[] publishTimestampByteArray = Longs.toByteArray(System.currentTimeMillis());
+            byte[] suffixByteArray = ("_" + baseMessage + "_" + i + 1).getBytes();
```

Comment:
```suggestion
            byte[] suffixByteArray = ("_" + baseMessage + "_" + (i + 1)).getBytes();
```

I think this is what you want.

Reply:
Updated